Digipost.Signature.Api.Client.Portal.Tests/Schema/Examples/Response/ErrorResponse.xml

```diff
@@ -1,4 +1,4 @@
-﻿<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
The good old magic BOM! A bug in Resharper introduced this. With the BOM, the file is not valid XML.